fbw-common/src/systems/instruments/src/EFB/Efb.tsx

```diff
@@ -399,6 +399,25 @@ export const Efb: React.FC<EfbProps> = ({ aircraftChecklistsProp }) => {
     }
   }, [nwStrgDisc]);
 
+  // Check if the aircraft is on the ground, tug not attached and stationary if not close all doors
+  // This uses all available interactive point indices to close all doors - on the A80X there are 20 doors.
+  // Triggering an interaction point that is not defined is ignored therefore we can also use this for the A32NX.
```

Comment:
I don't think we should do things that generate errors inside the sim intentionally.